### ISSUE TITLE
Change Dismiss infusion / spell to a swift action

### ISSUE THREAD

**Issue by highfly117** [Truinto/DarkCodex#328]
Currently Dismiss infusion / spell is a standard action this waste a characters whole round just to remove a spell.

I would like to change  change Dismiss infusion / spell so that it is a swift action but can't be done in the turn you cast the spell or infusion.

**Comment by Shodan14** [comment]
It's a standard action to dismiss a spell in Tabletop.

**Comment by FrijDom** [comment]
Yeah, this is a tabletop thing. It's partially to reduce the ability to dismiss spells mid-combat, as especially in p&p, it can be particularly OP since it's hard to tell when the next time they're allowed to act would be.
